<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkSaver</class>
 <widget class="QMainWindow" name="LinkSaver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LinkSaver</string>
  </property>
  <property name="windowIcon">
   <iconset resource="linkres.qrc">
    <normaloff>:/res/2globe_128.png</normaloff>:/res/2globe_128.png</iconset>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QTreeWidget" name="linkcat">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <attribute name="headerCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="headerHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="headerShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Links</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>235</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Category"/>
    <addaction name="actionAdd_App"/>
    <addaction name="actionADD"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionFrom_Firefox"/>
     <addaction name="actionFrom_Chrome"/>
     <addaction name="actionFrom_Xbel_Sax"/>
     <addaction name="actionFrom_Opera"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionXBel"/>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionADD">
   <property name="icon">
    <iconset resource="linkres.qrc">
     <normaloff>:/link</normaloff>:/link</iconset>
   </property>
   <property name="text">
    <string>Add Url</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Category">
   <property name="icon">
    <iconset resource="linkres.qrc">
     <normaloff>:/folder</normaloff>:/folder</iconset>
   </property>
   <property name="text">
    <string>Add Category</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="linkres.qrc">
     <normaloff>:/res/2globe_128.png</normaloff>:/res/2globe_128.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="RemoveFolderItem">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionAdd_App">
   <property name="icon">
    <iconset resource="linkres.qrc">
     <normaloff>:/appexec</normaloff>:/appexec</iconset>
   </property>
   <property name="text">
    <string>Add App</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="linkres.qrc">
     <normaloff>:/res/applications-accessories.png</normaloff>:/res/applications-accessories.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionFrom_Firefox">
   <property name="text">
    <string>From Firefox</string>
   </property>
  </action>
  <action name="actionFrom_Chrome">
   <property name="text">
    <string>From Chromium</string>
   </property>
  </action>
  <action name="actionXBel">
   <property name="text">
    <string>XBel</string>
   </property>
  </action>
  <action name="actionFrom_Xbel_Sax">
   <property name="text">
    <string>From Xbel(Sax)</string>
   </property>
  </action>
  <action name="actionFrom_Opera">
   <property name="text">
    <string>From Opera</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="linkres.qrc"/>
 </resources>
 <connections/>
</ui>
